src/components/CustomDrawer/CustomDrawer.js

```diff
@@ -50,8 +50,7 @@ const useStyles = makeStyles((theme) => ({
   },
   titleActionButton: {
     background: '#D8D8D8',
-    border: '1px solid #BABABA',
-    boxShadow: '0px 3px 4px rgba(0, 0, 0, 0.12)',
+    boxShadow: '0px 2px 4px rgba(0, 0, 0, 0.12)',
     borderRadius: '20px'
   }
```

Comment:
We should remove this here, the main goal of this feature is that we can only import the button and use it without create other styles.